build_msvc/vcpkg.json

```
@@ -13,5 +13,12 @@
       "features": ["thread"]
     },
     "zeromq"
+  ],
+  "builtin-baseline": "b86c0c35b88e2bf3557ff49dc831689c2f085090",
```

This baseline implies package versions as they were at `2022.02.23` tag, which means older ones than they are in our master branch for now. For details, see https://cirrus-ci.com/task/4695704203952128.

Suggesting to update it up to the `2023.01.09` tag:
```suggestion
  "builtin-baseline": "f14984af3738e69f197bf0e647a8dca12de92996",
```

---

done